ember-cli/2017-07/july-06.md

```diff
@@ -0,0 +1,11 @@
+# ember-cli Meeting Notes - 2017-07-06
+
+## Attendees
+
+
+
+## Agenda
```

Comment:
Move : https://github.com/ember-cli/rfcs/pull/108 to FCP (once I update the RFC for same name conflict to default to last one wins with warning only)